src/Spinner.java

```diff
@@ -0,0 +1,94 @@
+//Thrown 'Hierarchy of the type Spinner is inconsistent' uhhhhhh
+public interface Spinner<Slice>
+        extends Comparable<Spinner<Slice>>, SpinnerKernel1<Slice> {
+
+    //representation could be a Sequence of my custom
+    //Slice objects (literally just what Jeremy told me lol but that makes sense)
+
+    //All makes very much sense to me conceptually, but I def need to figure
+    //out how to implement my big kicker of spin properly.
+    //The way I am thinking in my head is a "weighted removeAny" type of thing
+    //This will be random, not arbitrary though, so I gotta
+    //a. figure out if random is allowed in something like this
+    //b. figure out how I would implement
+    //(if build on sequence, can access different indexes and randInt to pick)
+    //having a fun time though this project is really fun and cool!
+
+    interface Slice<key, value> {
```

Comment:
You can add weights to this object. In fact, there are a lot of ways to simulate weights. I would imagine that you would want the weights to be a fraction of 100, but they don't actually have to be (since you can treats weights as ratios of the whole). You can also populate an array with multiple of the same slice and randomly select from the array.